src/main/rx/crsf.c

```diff
@@ -25,7 +25,7 @@
 
 #include "platform.h"
 
-#ifdef USE_SERIAL_RX
+#if defined(USE_SERIAL_RX) && defined(USE_SERIALRX_CRSF)
```

Comment:
I think the checks for `defined(USE_SERIAL_RX)` should not be in here - considering that all of the `USE_SERIALRX_<protocol>` are undefined in `common_post.h` if `USE_SERIAL_RX` is not defined these are redundant, and will make maintenance harder.

Reply:
I agree.